### DISCUSSION TITLE
Calendar plugin bug.

### DISCUSSION THREAD

**Discussion by tzmosh** [Stellarium/stellarium#2152] [General]
In the Hebrew calendar included in the Stellarium, appeared yesterday (03/01), Hebrew date 30 Tevet.
In fact the correct date was 1 Shevat.
Today, the next day, the software skipped 1 Shevat, and the date listed is 2 Shevat.
(In fact in the Hebrew calendar this month Tevet contains only 29 days.)

**Comment by gzotti** [comment]
Moved to https://github.com/Stellarium/stellarium/issues/2153

**Comment by gzotti** [comment]
Solved. Thanks for notifying!

**Comment by tzmosh** [comment]
If I understand correctly, only in the next version can I notice a correction ?!
And as always I am very grateful to all the dedicated people behind Stellarium. This amazing project.

**Reply by alex-w** [reply]
Yes, the fix will be available in next version

**Reply by gzotti** [reply]
Should be in next beta, in case you need it earlier. But the date is past already...
